client/me/purchases/manage-purchase/auto-renew-toggle/index.jsx

```diff
@@ -219,7 +218,7 @@ class AutoRenewToggle extends Component {
 				'…'
 			) : (
 				<Button
-					link
+					isLink
```

Comment:
this raised the react error:
`Warning: React does not recognize the `isLink` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `islink` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at button
    at Button (app:///../packages/components/src/button/index.tsx:55:25)
    at AutoRenewToggle (app:///./me/purchases/manage-purchase/auto-renew-toggle/index.jsx:54:5)
    at eval (app:///../packages/i18n-calypso/src/localize.js:29:67)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at span
    at div
    at li
    at PurchaseMetaExpiration (app:///./me/purchases/manage-purchase/purchase-meta.jsx:504:3)
    at ul
    at PurchaseMeta (app:///./me/purchases/manage-purchase/purchase-meta.jsx:75:3)
    at div
    at Card (app:///../packages/components/src/card/index.tsx:19:3)
    at ManagePurchase (app:///./me/purchases/manage-purchase/index.jsx:148:5)
    at eval (app:///../packages/i18n-calypso/src/localize.js:29:67)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at main
    at Main (app:///./components/main/index.jsx:13:3)
    at PurchasesWrapper (app:///./me/purchases/controller.jsx:82:3)
    at eval (app:///../packages/i18n-calypso/src/localize.js:29:67)
    at div
    at div
    at div
    at Layout (app:///./layout/index.jsx:269:1)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at WithCurrentRoute (app:///./components/route/index.js:44:30)
    at MomentProvider (app:///./components/localized-moment/provider.js:27:5)
    at ConnectFunction (app:///../node_modules/react-redux/es/components/connectAdvanced.js:233:68)
    at Provider (app:///../node_modules/react-redux/es/components/Provider.js:18:20)
    at QueryClientProvider (app:///../node_modules/react-query/es/react/QueryClientProvider.js:38:21)
    at RouteProvider (app:///./components/route/index.js:22:3)
    at I18nProvider (app:///../node_modules/@wordpress/react-i18n/build-module/index.js:66:5)
    at LocaleProvider (app:///../packages/i18n-utils/src/locale-context.tsx:22:3)
    at CalypsoI18nProvider (app:///./components/calypso-i18n-provider/index.tsx:21:3)
    at ProviderWrappedLayout (app:///./controller/index.web.js:63:3)`